molr-mole-remote/src/main/java/io/molr/mole/remote/rest/RestRemoteMole.java

```diff
@@ -94,5 +94,11 @@ public void instructRoot(MissionHandle handle, StrandCommand command) {
     public void instructBlock(MissionHandle handle, String blockId, BlockCommand command) {
         clientUtils.post(MoleWebApi.instructBlockInstance(handle.id(), blockId, command.name()), MediaType.APPLICATION_JSON, BodyInserters.empty());        
     }
+    
+    @Override
+    public void instruct(final MissionHandle handle, final MissionCommand command) {
+        clientUtils.post(MoleWebApi.instructMission(handle.id(), command.name()), APPLICATION_STREAM_JSON,
```

Comment:
Why do we have here APPLICATION_STREAM_JSON and not APPLICATION_JSON as in the other cases?